db/crud.go

```diff
@@ -879,27 +879,21 @@ func (db *DatabaseCollectionWithUser) getAvailableRevAttachments(ctx context.Con
 
 // Moves a revision's ancestor's body out of the document object and into a separate db doc.
 func (db *DatabaseCollectionWithUser) backupAncestorRevs(ctx context.Context, doc *Document, newDoc *Document) {
-	newBodyBytes, err := newDoc.BodyBytes(ctx)
-	if err != nil {
-		base.WarnfCtx(ctx, "Error getting body bytes when backing up ancestor revs")
-		return
-	}
 
 	// Find an ancestor that still has JSON in the document:
 	var json []byte
 	ancestorRevId := newDoc.RevID
 	for {
 		if ancestorRevId = doc.History.getParent(ancestorRevId); ancestorRevId == "" {
-			// No ancestors with JSON found.  Check if we need to back up current rev for delta sync, then return
-			db.backupRevisionJSON(ctx, doc.ID, newDoc.RevID, "", newBodyBytes, nil, doc.Attachments)
+			// No ancestors with JSON found. Return early
```

Comment:
Why wouldn't we want to back up the current rev (as cv) in this case as well?

Reply:
Had to move the backing up of the current version of the doc to the function postWriteUpdateHLV as in this area here we don't actually have a cv to work with as we haven't macro expanded the cas into cv yet.